feat: add 0.74.8 testnet release notes (#969)
vegaprotocol/documentation: docs/releases/overview.md

### Bug fixes
- The `aggregationEpochs` API was returning the last 30 epochs that had results rather than the last 30 epochs overall. This has been fixed such that the response always starts from the current epoch and counts back the number of epochs given in the filter. This fix was implemented in the [issue 10722 ↗](https://github.com/vegaprotocol/vega/issues/10722).
- A snapshot bug was found when changing the SLA hysteresis network parameter in a batch proposal. This fix was implemented in the [issue 10743 ↗](https://github.com/vegaprotocol/vega/issues/10743).
- When LPs were voting on batch proposals they were not showing up in the API response. Batch proposal votes have been fixed to contain ELS per market and this now shows on the API. This fix was implemented in the [issue 10725 ↗](https://github.com/vegaprotocol/vega/issues/10725).
- It was possible to suspend a market via governance that was already in governance suspension. A fix has been implemented to prevent governance suspension of a market already suspended. This fix was implemented in the [issue 10744 ↗](https://github.com/vegaprotocol/vega/issues/10744).

> -  After a recent change to the ledger entries API it did not return data when filtering by transfer ID. The `transfer ID` is now associated with the ledger entry. This bug has been fixed in [issue 10374 ↗](https://github.com/vegaprotocol/vega/issues/10374).
> - Several bugs in isolated margin were resolved: 
>     - Cancelling order on entering auction [10750 ↗](https://github.com/vegaprotocol/vega/issues/10750)
>     -  - Fix position updates to undo positions changes on isolated margin failure. [10696 ↗](https://github.com/vegaprotocol/vega/issues/10696)

Reviewer: ```suggestion
- After a recent change to the ledger entries API it did not return data when filtering by transfer ID. The `transfer ID` is now associated with the ledger entry. This bug has been fixed in [issue 10374 ↗](https://github.com/vegaprotocol/vega/issues/10374).
- Several bugs in isolated margin were resolved: 
    - Cancelling order on entering auction [10750 ↗](https://github.com/vegaprotocol/vega/issues/10750)
    - Order margin was not being updated after an order was cancelled. [10696 ↗](https://github.com/vegaprotocol/vega/issues/10696)
```

- When requesting multiple party IDs using REST, the API reported one or more invalid parties, however when requesting them individually, the given party IDs are valid and results are returned. The API was refactored to support this use case. This bug has been fixed in [issue 10780 ↗](https://github.com/vegaprotocol/vega/issues/10780).

### API changes
- `list transfers request` query allows for optional `from account type` and `to account type` filtering
- `submit transfer`/`submit transfer proposal` commands and `list transfers` query now include optional `capRewardFeeMultiple`
- `new market proposal` and `update market proposal` commands and `list market` query now includes `tickSize` field
- `list votes` query has a new shape for equity-like share, `ELS per market`, which provides `market ID` and `els`

To review the changes in the last released version, see [here](https://github.com/vegaprotocol/vega/compare/v0.74.7...v0.75.0-preview.2).

> 
> 
> 
> 
> 
> 
> 
> 
> 
> 

Reviewer: ```suggestion
```
